test/release/index.js

```diff
@@ -111,36 +59,16 @@ const pages = [
         "key": "locate-user",
         "title": "Locate the user"
     },
-    {
-        "key": "extrusion-query",
-        "url": "./extrusion-query.html",
-        "title": "Fill extrusion querying with terrain"
-    },
     {
         "key": "3d-playground",
         "title": "3D Playground",
         "url": "./3d-playground.html"
     },
```

Comment:
I'd be in favor of splitting that one up in several more focused examples:
- Sky: toggle between gradient and atmosphere 
- Terrain: A default terrain page with a style switcher
- Free camera: A page utilizing the free camera API

The issue from this page is that the UI isn't really appropriate for toggling things on mobile. It was useful for v2 but in retrospect it's not clear what should be tested for this page as it's a bit too generic.